radix-engine/tests/test_runner.rs

```diff
@@ -0,0 +1,84 @@
+use radix_engine::ledger::*;
+use radix_engine::transaction::*;
+use scrypto::prelude::*;
+use radix_engine::model::Receipt;
+use radix_engine::model::Transaction;
+
+pub struct TestRunner<'l> {
+    executor: TransactionExecutor<'l, InMemorySubstateStore>,
+}
+
+impl<'l> TestRunner<'l> {
+    pub fn new(ledger: &'l mut InMemorySubstateStore) -> Self {
+        let executor = TransactionExecutor::new(ledger, true);
```

Comment:
Minor: I'd suggest we make `TestRunner` own an `InMemorySubstateStore`, and allow the constructor to take a parameter `trace` (I personally found it useful when writing tests).

Reply:
Was looking to do this initially but then ran into awkwardness with certain borrowed structures. Will probably fix this in a later refactor.